src/components/TextInput/BaseTextInput.js

```diff
@@ -287,26 +286,10 @@ class BaseTextInput extends Component {
                             </View>
                         </TouchableWithoutFeedback>
                     </View>
-                    {(!_.isEmpty(inputHelpText) || !_.isNull(this.props.maxLength)) && (
-                        <View
-                            style={[
-                                styles.mt1,
-                                styles.flexRow,
-                                styles.justifyContentBetween,
-                                styles.ph3,
-                            ]}
-                        >
-                            {!_.isEmpty(inputHelpText) && (
-                                <Text style={[formHelpStyles]}>{inputHelpText}</Text>
-                            )}
-                            {!_.isNull(this.props.maxLength) && (
-                                <Text style={[formHelpStyles, styles.flex1, styles.textAlignRight]}>
-                                    {this.state.value.length}
-                                    /
-                                    {this.props.maxLength}
-                                </Text>
-                            )}
-                        </View>
+                    {!_.isEmpty(this.props.errorText) && (
```

Comment:
@Tushu17 correct me if I'm wrong, but is it really necessary to perform this check?
Because `InlineErrorText` is doing this check for us, and returning null :)
```
if (_.isEmpty(props.children)) {
        return null;
}
```

Reply:
Yeah we don't need it, but I checked other places and that's how we had implemented this. Should I change it?